fflib/src/classes/fflib_QueryFactory.cls

```diff
@@ -759,7 +788,15 @@ public class fflib_QueryFactory { //No explicit sharing declaration - inherit fr
 			this.fieldName = fieldName;
 			this.setMessage( 'Invalid field \''+fieldName+'\' for object \''+objectType+'\'' );
 		}
-	}
+	}public class InvalidRelationshipException extends Exception{
```

Comment:
@squattingdog sorry for asking this when your PR is opened since so long ago. But could you clean format a bit? This class should be at least one line below the `}` And we usually add open brace below the line instead of beside. For instance 
`}public class InvalidRelationshipException extends Exception{`
should be 
`}
public class InvalidRelationshipException extends Exception
{`

Don't worry about the code it is not yours. We will merge it as soon as you format it a bit.

Reply:
yup, not sure how that happened...addressing those.